CONTRIBUTING.rst

```diff
@@ -392,11 +410,21 @@ example, if you're making a change to just the Java support, you might start
 looking for regressions by just running the ``test/Java/\*.py`` tests instead of
 running all tests with ``runtest.py -a``.
 
+- To actually submit the fix and any test work as a Pull Request,
+  there will be some version control steps. For example::
+
+      $ git checkout -b fix-1387
+      $ git modified     # check that this reports your expected list
+      $ git add `git modified`
+      $ git commit -s    # fill in a good description of your changes
+
+  And proceed to push the change as a PR.
+
 
 Building Packages
 =================
 
-We use SCons (version 3.1.2 or later) to build its own packages.  If you
+We use SCons (since version 3.1.2) to build its own packages.  If you
```

Comment:
actually not since. the current packaging logic for SCons should work with any scons newer than 3.1.2. 
SCons has always been used to package scons. (or at least for a very long time)

Reply:
Was there a different way you wanted that expressed then?

Reply:
I think the previous text is fine as is. or maybe instead of or later, `or newer` ?

Reply:
Okay, put it back (using newer).